suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -314,7 +316,17 @@ export const startDescriptorPreloadedDiscoveryThunk = createThunk(
             return;
         }
 
-        const supportedNetworks = selectDiscoverySupportedNetworks(getState(), areTestnetsEnabled);
+        let supportedNetworks = selectDiscoverySupportedNetworks(getState(), areTestnetsEnabled);
```

Comment:
Could this whole added part be part `selectDiscoverySupportedNetworks`?

Reply:
No, because that would affect the list I am using in dev utils to display all available options.

Reply:
I would need to create new selector. I think for this purpose it's fine.